src/nt_users_tool/constants.py

```diff
@@ -0,0 +1,47 @@
+from lib2to3.pgen2.token import NAME
+
+
+BLANK = " "
+SLASH = "/"
+
+USERNAME = "user_name"
```

Comment:
okay, I think I understand those constants, let's see if we can do something better :) Let's discuss it